src/rust/options/src/parse.rs

```diff
@@ -66,7 +66,7 @@ peg::parser! {
         // NB: ##method(X) is an undocumented peg feature expression that calls input.method(pos, X)
```

Comment:
This comment is now irrelevant and should be deleted.

Reply:
Deleted.